io_scene_niftools/nif_export.py

```diff
@@ -288,14 +243,10 @@ def execute(self):
             """
 
             # apply scale
+            data.roots = [root_block]
             scale_correction = bpy.context.scene.niftools_scene.scale_correction
             if abs(scale_correction) > NifOp.props.epsilon:
```

Comment:
💯

Reply:
Not sure when to use bpy.context.scene.niftools_scene.scale_correction and NifOp.props.scale_correction lol